Reverse_word_string.py

```diff
@@ -0,0 +1,21 @@
+// Reversing a string in Python
```

Comment:
Move this file to the correct directory as well.

Reply:
Could you please let me know in which directory should I place this?

Reply:
@Jahnavi-Jonnalagadda move it to string algorithms: https://github.com/OpenGenus/cosmos/tree/master/code/string_algorithms/src.

Reply:
Don't forget to reformat this file with black.

Reply:
How to reformat a file with black?

Reply:
@Jahnavi-Jonnalagadda either install it with `pip install` or use it online [here](https://black.now.sh/).

Reply:
Unable to install black. Could you please guide me.

Reply:
If you are unable to install it, use the online version. Just paste your code in the URL and put the modified code in this PR.

Reply:
Yes, made changes.